README.md

```diff
@@ -184,12 +186,55 @@ Note: For Windows, CMake or Zig can be used.
     cmake --build . --config Release
     ```
 
-1. Use Zig
+- Using Zig
 
     ```bash
     zig build -Drelease-fast
     ```
 
+#### BLAS Build
+
+Building the program with BLAS support may lead to some performance improvements in prompt processing using batch sizes higher than 32 (the default is 512). BLAS doesn't affect the prompt generation. There are currently three different implementations of it:
```

Comment:
"BLAS doesn't affect the prompt generation" seems wrong, something like "BLAS doesn't affect the normal generation performance" may be clearer.